src/test/test-mountpoint-util.c

```diff
@@ -298,8 +298,9 @@ TEST(is_mount_point_at) {
         assert_se(fd >= 0);
 
         assert_se(mkdtemp_malloc("/tmp/not-mounted-XXXXXX", &tmpdir) >= 0);
-        assert_se(is_mount_point_at(fd, basename(tmpdir), 0) == 0);
-        assert_se(is_mount_point_at(fd, strjoina(basename(tmpdir), "/"), 0) == 0);
+        ASSERT_OK(path_extract_filename(tmpdir, &tmpdir_basename));
+        ASSERT_OK_EQ(is_mount_point_at(fd, tmpdir_basename, 0), 0);
```

Comment:
ASSERT_OK_ZERO

Reply:
@awilfox I think Lennart is proposing using the macro at https://github.com/systemd/systemd/blob/a50fa2a40f4a91d49503d3588a3dd29ea05e559b/src/shared/tests.h#L263 FYI